guardian/media-atom-maker
# Update webpack to v3

## What does this change?

This PR updates the now-very-old Webpack v1 to Webpack v3, in preparation of upgrading to the current Webpack v4.

n order to make this upgrade, a couple of config changes were made to `webpack.conf.js` files to use the updated syntax.

After this, we should be able to upgrade to v4 more easily and address the numerous open dependabot PRs.

## How to test

Run the repository locally and see it spin up healthily. Or check out CODE which is healthy after deploying the branch.
